omod-common/src/main/java/org/openmrs/module/webservices/rest/web/api/RestService.java

```diff
@@ -27,62 +27,77 @@
 	/**
 	 * Parses a representation requested by the client via the http request
 	 * 
-	 * @param requested
+	 * @param requested representation
```

Comment:
the param is requested

Reply:
@HerbertYiga usually it is the explanation of what is expected to be stored in the parameter not necessarily the parameter name. So that whoever is reading the docs knows what to parse or expect from the method. 

The method Parses a representation requested by the client via the http request, so the parameter is `requested` but what is expected is a string representation. i.e full or default etc.